ydb/core/tx/datashard/upload_stats.h

```diff
@@ -78,8 +78,8 @@ struct TUploadRetryLimits {
 };
 
 struct TUploadLimits: TUploadRetryLimits {
-    ui64 BatchRowsLimit = 500;
-    ui64 BatchBytesLimit = 1u << 23; // 8MB
+    ui64 BatchRowsLimit = 50000;
+    ui64 BatchBytesLimit = 8388608; // 8MB
```

Comment:
maybe 2-4MB for prefix_kmeans? Because it commonly will use x2-3 memory, but in general up to x4 (level, prefix, posting, upload)